### PR TITLE
[cs] remove unused brackets, clean unused property assign, remove unused imports + sort them

### PR DESCRIPTION
**Pull request by TomasVotruba**
Follow-up to #14738

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Failing codeclimate are 2 suggestions to already existing code. Not related to change

File: libraries/classes/Header.php [hunk 1]
```diff
@@ -9,18 +9,7 @@
...
-use PhpMyAdmin\Config;
-use PhpMyAdmin\Console;
-use PhpMyAdmin\Core;
-use PhpMyAdmin\Menu;
-use PhpMyAdmin\Message;
 use PhpMyAdmin\Navigation\Navigation;
-use PhpMyAdmin\RecentFavoriteTable;
-use PhpMyAdmin\Sanitize;
-use PhpMyAdmin\Scripts;
-use PhpMyAdmin\Url;
```
Comment (williamdes) [inline libraries/classes/Header.php]: Maybe something needs to be fixed:
```
------ ---------------------------------------------------------------------- 
  Line   libraries/classes/Header.php                                          
 ------ ---------------------------------------------------------------------- 
  272    Class PhpMyAdmin\Url referenced with incorrect case: PhpMyAdmin\URL.  
 ------ ----------------------------------------------------------------------
```

File: libraries/classes/Controllers/Table/TableStructureController.php [hunk 2]
```diff
@@ -22,7 +22,6 @@
 use PhpMyAdmin\Sql;
 use PhpMyAdmin\SqlParser\Context;
 use PhpMyAdmin\SqlParser\Parser;
-use PhpMyAdmin\SqlParser\Statements\CreateStatement;
```
Comment (williamdes) [inline libraries/classes/Controllers/Table/TableStructureController.php]: Please add back this line, used by phpdoc !
Rebase on latest master :)
And run phpstan `./vendor/bin/phpstan analyse`
You will see why I think this line is needed or the phpdoc must be fixed

**Review by williamdes — CHANGES_REQUESTED** [review]

File: libraries/classes/Header.php (hunk 1 above)
Comment (TomasVotruba) [reply libraries/classes/Header.php]: Even if this is misscase, the `URL` is in the same namespace as this class, so it can be removed.

Comment (TomasVotruba) [reply libraries/classes/Header.php]: Fixed :+1:

File: libraries/classes/Controllers/Table/TableStructureController.php (hunk 2 above)
Comment (TomasVotruba) [reply libraries/classes/Controllers/Table/TableStructureController.php]: Fixed :+1:  + I've added PHPStan to Travis, so you don't have to report these errors manually.

**Comment by TomasVotruba** [comment]
Travis fails seem unrelated

**Review by williamdes — APPROVED** [review]

**Review by MauricioFauth — APPROVED** [review]

File: libraries/classes/DatabaseInterface.php [hunk 3]
```diff
@@ -170,9 +159,8 @@ public function query(
         int $options = 0,
         bool $cache_affected_rows = true
     ) {
-        $res = $this->tryQuery($query, $link, $options, $cache_affected_rows)
+        return $this->tryQuery($query, $link, $options, $cache_affected_rows)
             or Util::mysqlDie($this->getError($link), $query);
-        return $res;
```
Comment (mauriciofauth) [inline libraries/classes/DatabaseInterface.php]: This change seems to be causing the Travis failure.

Comment (TomasVotruba) [reply libraries/classes/DatabaseInterface.php]: Oh, I'm sorry about that

Comment (TomasVotruba) [reply libraries/classes/DatabaseInterface.php]: Is that really related to mariadb install?

https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/459818713#L89

Comment (TomasVotruba) [reply libraries/classes/DatabaseInterface.php]: Looks like `or` vs `||` behaves differently - [Stackoverflow](https://stackoverflow.com/a/5998330/1348344). I'll try to revert this change then to see if that fixes Travis

Comment (TomasVotruba) [reply libraries/classes/DatabaseInterface.php]: It's still failing: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/459841731#L89 :+1:

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Comment by TomasVotruba** [comment]
Thanks, in next PR I'll add tool for checking all this in CI, so you never have to do it again.

Thank you for such open and reactions. It's very motivating to contribute

**Comment by williamdes** [comment]
@TomasVotruba Thank you for contributing !

**Pull request merged**